ui/lib/apps/ContinuousProfiling/pages/List.tsx

```diff
@@ -75,32 +108,30 @@ export default function Page() {
             return (
               <Badge
                 status="error"
-                text={t('continuous_profiling.list.table.status.failed')}
+                text={t('conprof.list.table.status.failed')}
               />
             )
           } else if (rec.state === 'success') {
```

Comment:
Update the state after PR pingcap/ng-monitoring#50 is merged.

Reply:
Updated:

![image](https://user-images.githubusercontent.com/1284531/147309219-49f148cf-76be-4c5d-8529-33edaacc9945.png)